net-nds/389-ds-base/389-ds-base-1.4.2.16.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WANT_AUTOMAKE="1.16"
+
+inherit multilib flag-o-matic autotools systemd db-use
+
+DESCRIPTION="389 Directory Server (core librares and daemons)"
+HOMEPAGE="https://directory.fedoraproject.org/"
+SRC_URI="https://releases.pagure.org/${PN}/${P}.tar.bz2"
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="autobind auto-dn-suffix debug doc +pam-passthru +dna +ldapi +bitwise presence selinux systemd"
+
+COMMON_DEPEND="
+	>=sys-libs/db-4.2:=
```

Comment:
this should probably be 4.7 instead of 4.2.

https://pagure.io/389-ds-base/c/c4b896dc992c89022b6752006edc7604cf0fcee2
https://pagure.io/389-ds-base/blob/master/f/m4/db.m4#_106

Reply:
It should be db-5.3, we build and test against it in RHEL7, RHEL8 and Fedora for quite some time now.